crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,26 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
+    unreferenced_disks = Array.new
+    crowbar_wall[:claimed_disks].each do |disk, _claim|
+      path, device_name = disk.split("/")[-2..-1]
```

Comment:
Sorry, but I just realized that there is another issue with this.

IIRC there are Raid Controllers out there which make disks devices show up as:
/dev/cciss/<somefilename> (instead of /dev/sdX)

The current code will fails with those type of disks. What will make things more complicated is that such a disk will show up as "cciss![somefilename]" inside the ```node[:block_device]``` attributes :(

So in case ```crowbar_wall[:claimed_disks]``` contains just a disk we'd need to replace the ```/``` after cciss with ```!``` to check the ```node[:block_device]``` attributes.